imgui/core.pyx

```diff
@@ -1374,6 +1374,68 @@ def is_window_collapsed():
     return cimgui.IsWindowCollapsed()
 
 
+def set_tooltip(str text):
+    """Set tooltip under mouse-cursor.
+    Usually used with :func:`is_item_hovered()`.
+    For a complex tooltip window see :func:`begin_tooltip()`.
+
+    .. visual-example::
+        :auto_layout:
+        :height: 100
+        :width: 200
+        :click: 10 10
+
+
+        imgui.begin("Example: tooltip")
+        imgui.button("Click me!")
+        if imgui.is_item_hovered():
+            imgui.tooltip("Please?")
+        imgui.end()
+
+    .. wraps::
+        void SetTooltip(const char* fmt, ...)
+    """
+    # note: "%s" required for safety and to favor of Python string formating
+    cimgui.SetTooltip("%s", _bytes(text))
+
+
+def begin_tooltip():
+    """Use to create full-featured tooltip windows that aren't just text.
+
+    .. visual-example::
+        :auto_layout:
+        :height: 100
```

Comment:
You could resize the example canvas so whole tooltip will be displayed in docs. The:

        :height: 100
        :width: 200

should be fine